site/members.md

```diff
@@ -259,6 +261,11 @@ members:
 - email: gurek15033
   name: Gurek
 
+- email: kanika16047
```

Comment:
@peey 
This means Kanika's name is not in the mailing list ?
Is there any other way to check about the missing members data ?

Reply:
Yeah she's not on the mailing list data I had.

@kanikasaini are you on the byld mailing list yet?

If yes then it might mean that the data I have is old and I'll ask suryatej about how I could access the latest data